appIcons.js

```diff
@@ -1233,17 +1232,19 @@ function getInterestingWindows(windows, monitorIndex) {
         const showUrgent = settings.get_boolean('workspace-agnostic-urgent-windows');
         const activeWorkspace = global.workspace_manager.get_active_workspace_index();
         windows = windows.filter(function(w) {
-            return w.get_workspace().index() == activeWorkspace
-                || (showUrgent && (w.urgent || w.demandsAttention || w._manualUrgency));
+            const inWorkspace = w.get_workspace().index() == activeWorkspace;
```

Comment:
Since you're there, can you please change to `===` equality check?

Reply:
Will do !